Miniforge3/construct.yaml

```diff
@@ -29,7 +29,7 @@ user_requested_specs:
 {% if name.endswith("pypy3") %}
   - python 3.9.* *_pypy
   - conda >=24.7.1
-  - mamba >=1.5.9
+  - mamba >=1.5.11
```

Comment:
pypy stuff should be left alone.

Reply:
I have seen that it has been deprecated. Will it be removed on day?

Also, do notice that I am naively trying to update versions; but someone else might be more efficient performing this update.

Reply:
Jan 2025 i think is the plan